test/Prototypes/CollectionTransformers.swift

```diff
@@ -12,6 +12,9 @@
 // RUN: %target-run-stdlib-swift
 // REQUIRES: executable_test
 
+// FIXME: swift-3-indexing-model: This prototype needs to be updated to new indexing model
```

Comment:
I don't think silencing tests this way is helpful -- currently, we at least get a test failure.  We'd need to get back to it anyway.